### PR TITLE
[#435] fix bug : when diskoffering doesn't exist

### PR DESCRIPTION
**Pull request by charsyam**
reason:
KT ucloud starts to support "no disk model"

patch:
 when diskofferingid is not in response. set it to 0.

### CODE REVIEW

File: libcloud/compute/drivers/ktucloud.py [hunk 1]
```diff
@@ -55,8 +55,12 @@ def list_sizes(self, location=None):
         szs = self._sync_request('listAvailableProductTypes')
         sizes = []
         for sz in szs['producttypes']:
+            diskofferingid = 0
+            if 'diskofferingid' in sz:
+                diskofferingid = sz['diskofferingid']
```
Comment (Kami) [inline libcloud/compute/drivers/ktucloud.py]: `diskofferingid = sz.get('diskofferingid' , 0)` is nicer and more pythonic way to do that

**Comment by Kami** [comment]
Also needs a test case for a size without `diskofferingid` attribute.

**Comment by charsyam** [comment]
@Kami I added testcase and fixing lint issue :)

**Comment by Kami** [comment]
@charsyam Patch has been merged into trunk, this pr can be closed now.

Thanks!

**Pull request closed without merging**